FirebaseAuth/Tests/Unit/Fakes/FakeBackendRPCIssuer.swift

```diff
@@ -148,11 +148,10 @@ class FakeBackendRPCIssuer: NSObject, AuthBackendRPCIssuer {
       requestData = body
       // Use the real implementation so that the complete request can
       // be verified during testing.
-      Task {
-        self.completeRequest = await AuthBackend.request(withURL: requestURL!,
-                                                         contentType: contentType,
-                                                         requestConfiguration: request
-                                                           .requestConfiguration())
+      completeRequest = Task {
+        await AuthBackend.request(withURL: requestURL!,
```

Comment:
Before, assigning this property in the task led to it being resolved later than the method returns, causing for the need to sleep since it was assigned as a side effect of the `_ = try? await rpcImplementation.call(with: request)` call.  Now, the task is synchronously assigned to it's resolved value can be async-ly waited on later.